stable/postgresql/README.md

```diff
@@ -356,13 +356,13 @@ This way, the credentials will be available in all of the subcharts.
 It's necessary to specify the existing passwords while performing an upgrade to ensure the secrets are not updated with invalid randomly generated passwords. Remember to specify the existing values of the `postgresqlPassword` and `replication.password` parameters when upgrading the chart:
 
 ```bash
-$ helm upgrade my-release bitnami/influxdb \
-    --set postgresqlPassword=[POSTGRESQL_PASSWORD] \
-    --set replication.password=[REPLICATION_PASSWORD]
-```
-
-> Note: you need to substitute the placeholders _[POSTGRESQL_PASSWORD]_, and _[REPLICATION_PASSWORD]_ with the values obtained from instructions in the installation notes.
+$ export PGPASS=$(kubectl get secret --namespace default data-postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
```

Comment:
Namespace and secret can be set to other values. According to [_NOTES.txt_](https://github.com/helm/charts/blob/master/stable/postgresql/templates/NOTES.txt#L11), something like 
```suggestion
$ export POSTGRES_PASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "postgresql.secretName" . }} -o jsonpath="{.data.postgresql-password}" | base64 --decode)
```
is more generic